src/main/java/com/odysseusinc/arachne/portal/api/v1/dto/AnalysisCreateDTO.java

```diff
@@ -33,7 +33,7 @@
     @NotBlank
```

Comment:
Why not to add a similar message for title field rather than standard `May not be empty`?

Reply:
Done